CONTRIBUTING.md

```diff
@@ -151,7 +151,7 @@ pyenv install 2.7.11
 pyenv install 3.4.3
 pyenv install 3.5.0
 python setup.py install
-pyenv local 3.5.0 3.4.3 2.7.8 2.6.9
+pyenv local 3.5.0 3.4.3 2.7.11 2.6.9
```

Comment:
Would it make sense to change this to `2.7.*` ?

Reply:
That makes sense to me. There are actually some other issues with this file as well. I had a hard time getting set up and ended up just using Docker. I'd be happy to look a little closer at the areas I had issues with and propose some changes, if you think that would be useful.

Reply:
Yes, that would be absolutely amazing! Please do! 
Also, if you had any issues with Docker, please suggest changes there too. 

Our theory is that if you had issues, someone else is going to - so any time you see something, please let us know! We would much rather have 1000 PRs fixing small silly issues than 1 customer having trouble using the library. 

THANK YOU!!